snapshot_test.go

```diff
@@ -166,6 +179,29 @@ func TestSnapshotCreate(t *testing.T) {
 
 			for i, archive := range snapshot.Archives {
 				file1 := filepath.Join(targetdir, archive.Path)
+
+				// Check if the file is a (successfully) excluded one
+				isExcluded := false
+				for _, excl := range tt.excludesRestore {
+					excludedFile := filepath.Join(targetdir, excl)
+
+					if _, err := os.Stat(excludedFile); os.IsNotExist(err) {
+						if excludedFile == file1 {
+							isExcluded = true
+							break
+						}
+					} else if err != nil { // Some other error occured
+						t.Errorf("Failed to stat file %s: %v", excl, err)
+						return
+					} else { // File probably exists
+						t.Errorf("Failed to exclude file %s while decoding snapshot", excl)
+						return
```

Comment:
`continue` instead of `return`.

Reply:
We could drop this else block, as the above conditions either break or continue/return.